R/Report_KRI.R

```diff
@@ -56,21 +67,22 @@ Report_KRI <- function(
       GroupLevel <- gsub("[^[:alnum:]]", "", GroupLevel)
       SnapshotDate <- gsub("[^[:alnum:]]", "", as.character(SnapshotDate))
 
-      strOutpath <- paste0(getwd(), "/kri_report_", StudyID, "_", GroupLevel, "_", SnapshotDate, ".html")
+      strOutpath <- file.path(strOutputDir, paste0("kri_report_", StudyID, "_", GroupLevel, "_", SnapshotDate, ".html"))
```

Comment:
```suggestion
      strOutputFile<- paste0("kri_report_", StudyID, "_", GroupLevel, "_", SnapshotDate, ".html")
```

Or use `glue::glue("kri_report_{StudyID}_{GroupLevel}_{SnapshotDate}.html")` since I think we import it already.